src/terms

```diff
@@ -0,0 +1,89 @@
+# **[Terms of service](https://web3.storage/terms/#terms-of-service)**
```

Comment:
let headings be headings
```suggestion
# Terms of service
```

Reply:
I agree but I'm not sure if this is link is used elsewhere to reference/direct to this section.